### PR TITLE
Simplify arduino api

### PR DESCRIPTION
**Pull request by jaxxzer**

### CODE REVIEW

**Comment by jaxxzer** [comment]
`private` is already here https://github.com/bluerobotics/ping-protocol/pull/116/files#diff-3cca2ef226ff32a24a8340224ac602b5R115

**Comment by patrickelectric** [comment]
I just updated the title, since the generated API is not only used for arduinos

File: src/protocol/templates/ping1d.h.in [hunk 1]
```diff
@@ -154,19 +91,24 @@ public:
      * @param {{field.name}} - {% if field.units %}Units: {{field.units}}; {% endif %}{{field.description}}
 {% endfor %}
      *
-     * @return true when verify = false if a valid reply is received from the device.
-     * @return true when verify = true if a valid reply is received from the
+     * @return when verify = false: true if a valid reply is received from the device.
+     * @return when verify = true: true if a valid reply is received from the
```
[the remark this answers is not in the document]
Comment (jaxxzer) [reply src/protocol/templates/ping1d.h.in]: verify - means verify the response

do you have a suggestion to make it more clear?

[the remark this answers is not in the document]
Comment (jaxxzer) [reply src/protocol/templates/ping1d.h.in]: note `@return` does not appear at the html output.

**Pull request merged**